### DISCUSSION TITLE
Dynamic badges with dynamic colors

### DISCUSSION THREAD

**Discussion by ReenigneArcher** [badges/shields#9628] [Ideas] [answered]
Would it be possible to make the colors dynamic on dynamic badges?

Ideally this would be automatic based on the value (assuming the value is a number)

**Comment by chris48s** [comment, marked as answer]
Hello. It isn't possible to do this with the dynamic json/xml/yaml badges and we don't plan to add that feature to those badge.
If you want this level of customisation for a completely bespoke badge it can be done with the endpoint badge https://img.shields.io/badges/endpoint-badge this requires a bit more setup but it is much more customisable :)

**Reply by ReenigneArcher** [reply]
Awesome, thanks!
